src/main/resources/config-schema.yml

```diff
@@ -0,0 +1,23 @@
+type: object
```

Comment:
Is this file used anywhere, or is that coming later?

Reply:
It's coming later. I'm working on a `validate-config` subcommand for the CLI which will (eventually) use `config-schema.yml` to validate a project's `appmap.yml`.